geode-core/src/main/java/org/apache/geode/internal/tcp/TCPConduit.java

```diff
@@ -759,6 +771,8 @@ private ConnectionTable getConTable() {
 
   protected void basicAcceptConnection(Socket othersock) {
     try {
+      othersock.setSoTimeout(0);
```

Comment:
This change moves the SSL handshake to the Handshake thread, where it belongs.  This has caused backlog issues in production.